lib/views/mobile/receive/receive.dart

```diff
@@ -23,6 +23,7 @@ class Receive extends StatefulWidget {
 class _ReceiveState extends State<Receive> {
   String _code = '';
   String fileName = '';
+  int fileSize = 0;
```

Comment:
This code could be moved to shared base class as it will be copied between mobile/desktop otherwise.